pkg/parsers/parsers.go

```diff
@@ -37,6 +39,8 @@ func ParseHost(defaultTCPAddr, defaultUnixAddr, addr string) (string, error) {
 	}
 }
 
+// ParseUnixAddr parses and validates that the specified address is a valid unix address
+// and returns a formatted unix address, that may be used the specified default address
```

Comment:
The last part of this is a bit difficult to understand. I think I mistakenly gave you a bad recommendation here. Maybe "ParseUnixAddr parses and validates that the specified address is a valid UNIX socket address. It returns a formatted UNIX socket address, either using the address parsed from addr, or the contents of defaultAddr if addr is a blank string".

Totally unrelated to this PR, but ParseHost has a silly line:

    addr = fmt.Sprintf("%s", defaultTCPAddr)